src/examples/subsidy.ts

```diff
@@ -0,0 +1,104 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import { Identity, PermissionedAccount } from '@polymeshassociation/polymesh-sdk/types';
+
+import { getClient } from '~/common/client';
+import { parseArgs } from '~/common/utils';
+
+/*
+  This script showcases how to interact with Subsidy, it
+  - grants a subsidy to an account
+  - increases allowance
+  - decreases allowance
+  - sets allowance
+  - quits subsidy
+
+  Usage e.g: yarn run-example ./src/examples/subsidy.ts account=ACCOUNT_ADDRESS
+*/
+(async (): Promise<void> => {
+  const { account: beneficiary } = parseArgs<ScriptArgs>(process.argv.slice(2));
+
+  if (!beneficiary) {
+    throw new Error('Please supply an account as an argument to the script');
+  }
+
+  console.log('Connecting to the node...\n');
+
+  const api = await getClient(process.env.ACCOUNT_SEED);
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const identity = (await api.getSigningIdentity())!;
+  console.log(`Connected! Signing Identity ID: ${identity.did}`);
+
+  // get subsidizer account
+  const { account: subsidizer } = await identity.getPrimaryAccount();
+
+  // granting a subsidy to beneficary
+  const subsidyGrantQ = await api.accountManagement.subsidizeAccount({
+    beneficiary,
+    allowance: new BigNumber(1000),
+  });
+  const subsidyGrantResult = await subsidyGrantQ.run();
+
+  console.log(`Subsidy request has been created and will expire in ${subsidyGrantResult.expiry}`);
+
+  // beneficiary now has to accept the subsidy request
+  // it is possible to run accept method on `subsidyGrantResult` however, it is more realistic to store the `authId` and then ask the beneficiary to accept the subsidy
+  // here as example we use BOB_SEED as the account that would accept the subsidy, in real life this would be signed by the actual beneficiary
+  const beneficiaryApi = await getClient(process.env.BOB_SEED);
+
+  const beneficiaryIdentity = (await beneficiaryApi.getSigningIdentity()) as Identity;
+
+  const {
+    account: beneficiaryAccount,
+  } = (await beneficiaryIdentity.getPrimaryAccount()) as PermissionedAccount;
+
+  const authorization = await identity.authorizations.getOne({ id: subsidyGrantResult.authId });
+  const acceptTx = await authorization.accept({ signingAccount: beneficiaryAccount });
```

Comment:
I was thinking you were gong to use one API, with multiple accounts loaded in.

since you made another API, use the `beneficiaryIdentity` and the resulting transactions will have a Context that has the intended default signer so this doesn't need to be specified.